src/vite.rs

```diff
@@ -46,6 +47,7 @@ pub struct Development {
 impl Default for Development {
     fn default() -> Self {
         Development {
+            base: "/",
```

Comment:
Can we make this an empty string by default, and require that users include the preceding slash but not the following (e.g., `.base("/app")`? I'm looking at `vite-ruby` as an example here: https://vite-ruby.netlify.app/config/#base

Reply:
Make more sense 👍